### DISCUSSION TITLE
forgot password

### DISCUSSION THREAD

**Discussion by lantura** [mealie-recipes/mealie#1039] [Q&A] [answered]
Hi,
I forgot my admin password for mealie and the defaults don't work.
No chance to login.
Any hints for reset admin password?
Thanks

**Comment by zackbcom** [comment]
Could download the sqllite db and edit the password if you know how.

**Reply by lantura** [reply]
Have edited & saved the password in user table, but login don't work ...
Any further hints?
Thanks.

**Reply by hay-kot** [reply]
@lantura, the passwords are stored as a hash. You need to replace the password in the database with the hash value of the password, not the plain text.

**Comment by lantura** [comment]
Can no one help with this topic?
Thanks.

**Comment by gustaf-e** [comment]
I did the following successfully:
1. Open the database with sqlite3
2. Hash your new password with Bcrypt
3. Update the users table `update users set password = 'HASHED_PASSWORD' where id = YOUR_USER_ID;`

**Comment by Trevo525** [comment, marked as answer]
There is now an official guide. When I googled this, I was linked this issue instead of the official guide. So, i am just leaving a link for someone in the future.

https://docs.mealie.io/documentation/getting-started/faq/#how-can-i-unlock-my-account
